### PR TITLE
4 landing page mobile

### LINKED ISSUE #4: Create Landing Page( Html+Css) Mobile
Please Note:
1. Write the Landing Page code including HTML, CSS, JS.
2. The landing page layout should be mobile first (Iphone 13 Pro Max)
3. The colors and fonts as in the figma file.

### PR DESCRIPTION
**Pull request by salwasiksik**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,26 @@
   <link rel="stylesheet" href="styles/landing.css">
 </head>
 <body>
+<!-- logo -->
+<div class="logo">
+  <img class="pic-of-logo" src="../assets/images/landing-page/logo.png">
```
Comment (azzsal) [inline index.html:17]: remove the ".." from here.

File: index.html [hunk 2]
```diff
@@ -12,6 +12,26 @@
...
+</div>
+<!-- slogo -->
+<div class="para-pf-slogo">
+<p class="slogo">Vacation is not at home,<br> discover the world with us</p>
+</div>
+</header>
+<!-- add the travel photo -->
+<main>
+  <img class="travel-pic" src="../assets/images/landing-page/travel.png">
```
Comment (azzsal) [inline index.html:26]: from here as well.

**Review by ahmedhmmad — CHANGES_REQUESTED** [review]
1- You need to add dot before CSS link file <link rel="stylesheet" href="styles/global.css"> ==>( ./styles/)
2- most of CSS styles are not applied on my end due to this issue

**Review by ahmedhmmad — APPROVED** [review]

**Review by ahmedhmmad — APPROVED** [review]

File: styles/landing.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik&display=swap");
```
Comment (azzsal) [inline styles/landing.css:1]: No need to import the font. It's already in the global.css file.

**Review by azzsal — APPROVED** [review]

**Review by Yousef-Dall — APPROVED** [review]

**Pull request merged**